### PR TITLE
Feature/readme documentation

### LINKED ISSUE #59: README and Documentation
- [x] Code Documentation
- [ ] Kakapo description
- [ ] Update README
- [ ] README.playground (?)
- [ ] podspec description
- [x] Discuss for devlucky.github.io

### PR DESCRIPTION
**Pull request by joanromano**
Closes #59 (or attempts to)

Creating a first draft for the README so we have a place to discuss while live seeing the actual result. I expect this to be a place where we come up with the better solution for our README in regards of structure and clear explanation of the needed concepts, as well as being a great source for being able to use basic functionalities of the library in a quick way. 

I strongly believe this shouldn't be a place where we 100% explain all implementation details and possibilities of the library, but I also may be wrong on this assumption. Please note this is a draft and everything is 100% flexible to be changed, so nothing here is final.

Apart from the actual structure of the file itself, some things missing that I am not 100% sure whether we should include a explanation for them here:
- [x] CustomSerializable
- [x] PropertyPolicy
- [x] SerializationTransformer (didn't put it because couldn't find good words for it yet, but should be in I believe)

### CODE REVIEW

File: Source/KakapoDB.swift [hunk 1]
```diff
@@ -29,7 +29,7 @@ public protocol _Storable {
 }
 
 /**
- A protocol that supports both `_Storable` and `Equatable` objects, handling an `id` and a generic `init` method, as well as value equality. 
+ A protocol that supports `_Storable` objects, handling an `id` and a generic `init` method.
```
Comment (joanromano) [inline Source/KakapoDB.swift]: @MP0w we should probably get rid of `_Storable`?

Comment (MP0w) [inline Source/KakapoDB.swift]: true, done in #92

**Comment by MP0w** [comment]
Cool, there are some part I want to rephrase (will open PR), we should mention kakapo.js, that we are looking for feedbacks and question since it's a v0.1 and the "donate to Kakapo" link!

**Comment by MP0w** [comment]
You can merge it already since it's better than the empty one

**Pull request merged**